incubator/hnc/cmd/manager/main.go

```diff
@@ -84,11 +88,23 @@ func main() {
 	defer exporter.Flush()
 
 	if err := exporter.StartMetricsExporter(); err != nil {
-		setupLog.Error(err, "cannot start metric exporter")
+		setupLog.Error(err, "cannot start StackDriver metric exporter")
 		os.Exit(1)
 	}
 	defer exporter.StopMetricsExporter()
 
+	promExporter, err := prometheus.NewExporter(prometheus.Options{Registry: metrics.Registry})
```

Comment:
Hi, @adrianludwin 
Here I use the metrics registry from controller-runtime, and uses it to serve the metrics from open census. And then seve it. That's it

Reply:
Thanks, that's really interesting. But how does this conflict with the existing endpoint? You can't have two servers in the same process serving on the same port. Perhaps there's some kind of change you need to make in `/config/...` to disable the builtin one?

This is why I'd like to know that it actually works if you test it.

Reply:
Yes, I remove that part. That's my miss. I think maybe it is work now. But I don't have an environment to test it

Reply:
We can test the Stackdriver part. But we don't have the time or even expertise to test the Prometheus portion ourselves right now. Do you not have access to a GKE (or Azure or EKS) cluster? I know GCP will give you $300 in free credits, of which you should only have to spend a couple of dollars to spin up a small cluster for a day or two to test this.

Unfortunately we can't accept PRs without testing, and the testing is possibly the hardest part of this change.

Reply:
OK. I will setup the cluster in my own and test it